README.md

```diff
@@ -167,7 +167,12 @@ The values to configure are:
 The domains used in the aliases must have been previously created in TMail.
 Otherwise, if a user have a single alias pointing to an unknown domain, none of her aliases will be added.
 
-The jar of the TMail LSC plugin must be copied in the `lib` directory of your LSC installation.
+For the domain synchronization, you can specify the wished domain list to be synchronized in `SyncContactConfig` file.
+If you specify Optional.empty, all domains contact will be synchronized from LDAP.
+
+If you want to change the wished domain list in `SyncContactConfig` which will break the unit tests, please compile the project using the command: `mvn clean install -DskipTests`.
+
+The jar of the TMail LSC plugin (`target/lsc-tmail-plugin-1.0-distribution.jar`) must be copied in the `lib` directory of your LSC installation.
```

Comment:
IMO We will hard code the wished domain list in the tmail-lsc jar anyway so specifying the config in a resource file does not help much better so I came up with a quick solution.

Reply:
Can we use, say ENV variable instead of recompiling?